lib/_007/Parser/Syntax.pm

```diff
@@ -170,6 +170,35 @@ grammar _007::Parser::Syntax {
 
     token str { '"' ([<-["]> | '\\\\' | '\\"']*) '"' } # " you are welcome vim
 
+    rule regex-part {
+        <regex-group>
+        [ '|' <regex-group> ]*
+    }
+
+    rule regex-group {
+        <regex-quantified> +
+    }
+
+    token regex-quantified {
+        <regex-fragment> $<quantifier>=<[+ * ?]>?
+    }
+
+    proto token regex-fragment {*}
+    token regex-fragment:str {
+        <str>
+    }
+    token regex-fragment:identifier {
+        <identifier>
+    }
+    token regex-fragment:call {
+        '<' ~ '>'
+        <identifier>
+    }
+    token regex-fragment:group {
+        '[' ~ ']'
+        <regex-part>
```

Comment:
Not sure why, but

    $ bin/007 -e='/["bc"]/'

works, whereas

    $ bin/007 -e='/ ["bc"]/'
    Could not parse program

(with an extra space before the `[`) fails. In my view it should parse just the same.

Reply:
>     $ bin/007 -e='/ ["bc"]/'
>     Could not parse program

This still fails (and in the same way).

Reply:
I'll add the needed `<.ws>` in my next commit, along with more parsing tests (especially for search and well-formedness).